src/main/resources/com/google/api/codegen/csharp/wrapper.snip

```diff
@@ -190,13 +190,13 @@
             /// </remarks>
             public CallSettings {@method.grpcName}Settings { get; set; } = new CallSettings
             {
-                RetrySettings = new RetrySettings
+                Timing = CallTiming.FromRetry(new RetrySettings
```

Comment:
So are we later going to add code that *doesn't* call FromRetry when there are no retries to make?